boards/arm64/qemu_cortex_a53/qemu_cortex_a53_defconfig

```diff
@@ -1,7 +1,9 @@
 CONFIG_SOC_QEMU_CORTEX_A53=y
 CONFIG_BOARD_QEMU_CORTEX_A53=y
 CONFIG_ARM_ARCH_TIMER=y
-CONFIG_QEMU_ICOUNT=n
+CONFIG_QEMU_ICOUNT=y
+CONFIG_QEMU_ICOUNT_SHIFT=4
+CONFIG_QEMU_ICOUNT_SLEEP=y
```

Comment:
It looks like icount was deliberately disabled for this platform:
https://github.com/zephyrproject-rtos/zephyr/commit/1be06335fa3bc0ad7655aff508950d62254b9f71

@carlocaione Is the problem mentioned in the above commit still relevant?

Also should https://github.com/zephyrproject-rtos/zephyr/pull/32049/commits/eadc73bfbe60284fcbe34432f472869869628528 be brought in again?

Reply:
It was removed for performance reasons but I guess fixing the CI issues is more important.